app/controllers/questions_controller.rb

```diff
@@ -5,6 +5,11 @@ class QuestionsController < ApplicationController
   # GET /questions.json
   def index
     @questions = Question.all
+    if params[:search]
+      @questions = Question.search(params[:search]).order("created_at DESC")
```

Comment:
`Question.search` doesn't seem to be an actual method. Since we've implemented search with datatables already, I would recommend removing this for now.